src/main/java/org/opentripplanner/standalone/server/OTPWebApplication.java

```diff
@@ -115,6 +117,24 @@ public Map<String, Object> getProperties() {
     return props;
   }
 
+  /**
+   * Return an HK2 Binder that injects this specific OtpServerContext instance into Jersey web
+   * resources. This should be registered in the ResourceConfig (Jersey) or Application (JAX-RS) as
+   * a singleton. Jersey forces us to use injection to get application context into HTTP method
+   * handlers, but in OTP we always just inject this OTPServer instance and grab anything else we
+   * need (routers, graphs, application components) from this single object.
```

Comment:
This refers to old concepts

Reply:
I changed "always just inject this OTPServer instance and grab anything" to "always just inject this OtpServerContext and grab anything". The rest is ok?